~/.flightctl/config.yaml

```
@@ -0,0 +1,35 @@
+  hostname: 127.0.0.1
+  port: 6379
+  password: DGE06-iyvNc-ahvwW-srnN5
+service:
+  address: :2443
+  agentEndpointAddress: :6444
+  agentGrpcAddress: :7444
+  baseAgentEndpointUrl: https://localhost:6444
+  baseAgentGrpcUrl: grpcs://localhost:7444
+  baseUrl: https://localhost:2443
+  cert: /home/abennata/.flightctl/certs
+  logLevel: info 
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)